### DISCUSSION TITLE
Environment variable PGDATA=/tmp in docker-compose.yml

### DISCUSSION THREAD

**Discussion by Trapper-M** [dyc3/opentogethertube#1817] [Q&A] [answered]
I succesfully installed OTT via docker following the instructions provided in the deployment docs.
Everything worked fine. But after I stopped and removed the containers and started new ones I recognized that all data (Users, permanent rooms) wasn't available anymore.
After creating some new users and rooms I checked the postgres container and found that the database is not stored in the volume db-data-postgress but in folder /tmp inside the container.
After I have removed the statement PGDATA=/tmp from the docker-compose.yml file the data was stored in the path /var/lib/postgresql/data which is assigned to the volume.

Is there any reason why the environment variable PGDATA should be set to /tmp?

**Comment by dyc3** [comment, marked as answer]
The original docker stuff was contributed years ago and the reason for including that line is probably lost to time. I'll go ahead and fix it. #1818
